compiler/circle-opselector/driver/Driver.cpp

```diff
@@ -153,6 +155,107 @@ std::vector<std::string> split_name_input(const std::string &str)
   return ::split_into_vector(str, ',');
 }
 
+std::unique_ptr<loco::Graph> make_graph(const std::vector<const luci::CircleNode *> nodes)
+{
+  auto graph = loco::make_graph();
+
+  luci::CloneContext ctx;
+  // clone nodes
+  for (const auto &n : nodes)
+  {
+    auto clone = luci::clone_node(n, graph.get());
+    ctx.emplace(n, clone);
+  }
+  // set graph input
+  for (const auto &n : nodes)
+  {
+    for (uint32_t i = 0; i < n->arity(); i++)
+    {
+      auto arg = n->arg(i);
+      auto input_node = dynamic_cast<luci::CircleNode *>(arg);
+      auto ctx_it = ctx.find(input_node);
+      // check if the node already has been cloned
+      if (ctx_it != ctx.end())
+        continue;
+      // the node isn't graph input if it is an other node's input
+      if (std::find(nodes.begin(), nodes.end(), arg) != nodes.end())
+        continue;
+      auto circle_const = dynamic_cast<luci::CircleConst *>(arg);
+      if (circle_const == nullptr)
```

Comment:
IMHO it would better use like `if (circle_const != nullptr) { ... }` for `CircleConst`